WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderPostUiStateBuilder.kt

```diff
@@ -122,6 +125,10 @@ class ReaderPostUiStateBuilder @Inject constructor(
             post.takeIf { post.cardType == VIDEO }
                     ?.let { post.featuredVideo }
 
+    private fun buildExpandedTagsViewVisibility(post: ReaderPost) = post.isDiscoverPost
+
+    private fun buildTags(post: ReaderPost) = ReaderTagList() // TODO: ashiagr - get tags from post
```

Comment:
Topics on the card are currently set to empty tags list and needs to set from the post.

Reply:
Done here: 63c9d8d